data/json/itemgroups/Locations_MapExtras/mall_item_groups.json

```diff
@@ -96,6 +96,8 @@
       [ "folded_bicycle", 5 ],
       [ "sunglasses", 25 ],
       [ "cowboy_hat", 25 ],
+      { "item": "mp3", "prob": 40, "charges": 100 },
```

Comment:
And here?

Reply:
I don't remember tinkering with the mp3 item at any point while making these fursuit variants, so I don't think these need any editing (apologies if I'm missing anything - and If I did, please tell me and I'll try my best to remedy it, still pretty new to this!)

Reply:
checked the files of this pull request to see if there were extra mp3 players, but i didn't find any

Reply:
Looks like git was doing some weird stuff. Maybe close this PR and create a new one?